.github/workflows/chromatic.yml

```diff
@@ -34,6 +43,23 @@ jobs:
     environment: staging
     # Job steps
     steps:
+      # Determine if the PR comment should trigger the e2e test suite
+      - name: Check if user is part of Isomer core team
+        uses: tspascoal/get-user-teams-membership@v1
+        id: checkUserMember
+        with:
+          username: ${{ github.actor }}
+          team: ${{ env.ISOMER_CORE_TEAM_SLUG }}
+          GITHUB_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }} # requires read:org
+
+      - name: Check for trigger words (in PR comment)
```

Comment:
i think you missed out the step where this triggers the workflow - see [here](https://github.com/Khan/pull-request-comment-trigger)

what we need is probably (in the next step): 
```
      - run: 'echo Found it!'
        if: steps.check.outputs.triggered == 'true'
```

Reply:
Hey, isnt this checked in line 42?

Reply:
As discussed offline, 20d05f4d81ae33e983dd72ecfea8d8be3be42ff3. Thanks for the catch!